DockerFile

```diff
@@ -0,0 +1,46 @@
+FROM python:3.8-slim-buster
+
+RUN apt-get update
+RUN apt-get -y install gcc
+
+WORKDIR /rest/BackendApp
+
+COPY ./requirements/base.txt ./
+COPY ./requirements/local.txt ./
+COPY ./requirements/production.txt ./ 
+COPY ./requirements/staging.txt ./
+
+# Prevents Python from writing pyc files to disc
+ENV PYTHONDONTWRITEBYTECODE 1
+
+# Prevents Python from buffering stdout and stderr
+ENV PYTHONUNBUFFERED 1
+
+
+RUN pwd
+
+RUN ls -l
+
+RUN ls -a
+
+COPY ./docker-build/wsgi-entry.sh /rest/
+
+RUN chmod +x /rest/wsgi-entry.sh 
+
+RUN  apt-get install -y --no-install-recommends python-dev libpq-dev
+
+RUN mkdir -p /rest/logs && \
+    pip3 install wheel==0.34.2 && \
+    pip3 install -r base.txt 
+#    pip3 install -r local.txt && \
+#    pip3 install -r production.txt && \
+#    pip3 install -r staging.txt 
+
+RUN ls -a
+
+COPY ./ ./
+
+CMD [ "python", "manage.py",  "migrate"]
+CMD [ "python", "manage.py",  "runserver", "0.0.0.0:8000" ]
```

Comment:
we should go with  gunicorn or anyother wsgi sapi for production...instead of runserver

Reply:
we can commet this out, it is currently running thru gunicorn only.